### ISSUE TITLE
The color contrast of a few components on the front page isn't meeting WCAG2.1 SC 1.4.3 & 1.4.11

### ISSUE THREAD

**Issue by rpkoller** [mastodon/mastodon#19870]
### Steps to reproduce the problem

1. go to your homepage (in my case https://mastodon.social/home)
2.take a look at the text and the icons

### Expected behaviour

The color contrast of a few text components and icons should be higher

### Actual behaviour

The actual color contrast is too low. 

### Detailed description

I've checked the main page of my account on https://mastodon.social/home

**WCAG 2.1 SC 1.4.3.**  
_`Footer` & `trending now stats subline` & `status card host`:_ 

Text color : `#606984` 
Background color: `#313543`    
color contrast `2.23:1` (with the current font-size text should have at least  `4,5:1`. `3:1`  in case font-size is `>= 24px` or `>=18.5px` if it is also `bold` )

_`status display name` & `time status of the visibility icon`:_ 
text color: `#606984`
Background: `#3b3e48`   
color contrast `1.95:1`  (with the current font-size text should have at least  `4,5:1`. `3:1`  in case font-size is `>= 24px` or `>=18.5px` if it is also `bold`) 

**WCAG2.1 SC 1.4.11** 
_`action icon on each toot in the timeline` & `the visibility icon on each toot in the timeline`_

icon color: `#71788f`
background color : `#3b3e48`
color contrast: `2.43:1`  (should have at least `3:1`) 

### Specifications

 Mastodon v.4.0.0rc1 (on https://mastodon.social/home)

Tested with Safari 16.1 and Edge 107.0.1418.35 on MacOS Monterey

**Comment by diondiondion** [comment]
This has been fixed on https://mastodon.social and https://mastodon.online, and the change will be part of Mastodon 4.6
